adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/unicredit/UnicreditAccountInformationService.java

```diff
@@ -60,6 +60,13 @@ public UnicreditAccountInformationService(Aspsp aspsp,
         return super.getAccountList(requestHeaders, RequestParams.builder().build());
     }
 
+    @Override
+    public Response<OK200AccountDetails> getAccountDetails(String accountId,
+                                                           RequestHeaders requestHeaders,
+                                                           RequestParams requestParams) {
+        return super.getAccountDetails(accountId, requestHeaders, RequestParams.builder().build());
```

Comment:
Why RequestParams should be reset here?

Reply:
I implemented it analogously to "Read Account List". The only query param for these endpoints is "withBalance".

According to the UniCredit API documentation this query param will be ignored for "Read Account List" and "Read Account Details".

I just did a test with the query param "withBalance" (true and false) and got no error for "Read Account List" and "Read Account Details". I think it is not necessary to reset the RequestParams for both endpoints.

Reply:
Good point!
`readAccountList` shouldn't have a RequestParams reset, it was implemented that way with no clear reason and will be fixed in the future.

I'd recommend not overriding `readAccountDetails` here and omit passing empty RequestParams

Also, please update this branch with the latest Adapter commits from the develop one, you're missing some stuff

Reply:
OK :+1: I removed this method and updated our fork.